.github/workflows/build.yml

```diff
@@ -54,8 +55,6 @@ jobs:
           git \
           java-11-openjdk-devel \
           make \
-          maven-profile \
-          maven-plugin-registry \
           maven-source-plugin \
           maven-shade-plugin \
```

Comment:
maven-shade-plugin is in javapackages-tools module n PowerTools/CRB, please take a look at below:

https://github.com/oVirt/ovirt-engine-extensions-api/blob/master/.github/workflows/build.yml#L38
https://github.com/oVirt/ovirt-engine-extensions-api/blob/master/.github/workflows/build.yml#L51

Reply:
Enabling powertools and modules for el8 solved the issue. EL9 has been fine anyway.